File: archive/boj/2562/ppoddo/최댓값.py
import sys

def main() -> None:
    input = sys.stdin.readline
    nums =[int(sys.stdin.readline().strip()) for _ in range(9)]
    max_value = max(nums)
    idx = nums.index(max_value) + 1 # index + 1
    print(max_value)
    print(idx)

if __name__ == "__main__":
    main()

File: archive/boj/2562/bluehalu98/최댓값.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\n/);

let idx = 0;
let max = +input[0];

for (let i = 1; i < 9; i++) {
  if (max < +input[i]) {
    max = +input[i];
    idx = i;
  }
}

console.log(`${max}\n${idx + 1}`);